02-peer-protocol.md

```diff
@@ -1116,6 +1116,9 @@ This message initiates the v2 channel establishment workflow.
    1. type: 1 (`channel_type`)
    2. data:
         * [`...*byte`:`type`]
+   1. type: 2 (`require_confirmed_inputs`)
+   2. data:
+        * [`0*byte`:``]
```

Comment:
Hm i think there's a different way to do this..

Reply:
Ah I see, it's a zero-length TLV item.

Reply:
Let me check in with the parser.. i'm not sure we have an annotation for that yet. Do you know if this is the first one in the spec?

Reply:
It is indeed the first one in the spec...